File: GalleryStudio/Customers.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GalleryStudio
{
    [Serializable]
    public class Customers
    {
        static Int32 NumberOfObjects = 0;
        const char NULL='\0';
    
        Int32  CustomerID = new Int32();
        String Name = new String(NULL,20);
        String FamilyName = new String(NULL,20);
        String Address = new String(NULL,20);
        Int32  TelephonNumber = new Int32();

        public Customers()
        {
        
        }
        public Int32 CustomenrNumberOfObjectsAccess
        {
            get {
                return NumberOfObjects;
            }
            set
            {
                NumberOfObjects = value;
            }
        }
        public Int32 CustomersCustomerID
        {
            get
            {
                return CustomerID;
            }
            set
            {
                CustomerID = value;
            }
        }
        public String CustomersName
        {
            get
            {
                return Name;
            }
            set
            {
                Name = value;
            }
        }
        public String CustomersFamilyName
        {
            get
            {
                return FamilyName;
            }
            set
            {
                FamilyName = value;
            }
        }
        public String CustomersAddress
        {
            get
            {
                return Address;
            }
            set
            {
                Address = value;
            }
        }
        public Int32 CustomersTellefon
        {
            get
            {
                return TelephonNumber;
            }
            set
            {
                TelephonNumber = value;
            }
        }
        
    }
}

File: refrigitz 12/Customers.cpp
#include "Customers.h"


namespace GalleryStudio
{

int Customers::NumberOfObjects = 0;

	Customers::Customers()
	{

		InitializeInstanceFields();
	}

	const int &Customers::getCustomenrNumberOfObjectsAccess() const
	{
		return NumberOfObjects;
	}

	void Customers::setCustomenrNumberOfObjectsAccess(const int &value)
	{
		NumberOfObjects = value;
	}

	const int &Customers::getCustomersCustomerID() const
	{
		return CustomerID;
	}

	void Customers::setCustomersCustomerID(const int &value)
	{
		CustomerID = value;
	}

	const std::wstring &Customers::getCustomersName() const
	{
		return Name;
	}

	void Customers::setCustomersName(const std::wstring &value)
	{
		Name = value;
	}

	const std::wstring &Customers::getCustomersFamilyName() const
	{
		return FamilyName;
	}

	void Customers::setCustomersFamilyName(const std::wstring &value)
	{
		FamilyName = value;
	}

	const std::wstring &Customers::getCustomersAddress() const
	{
		return Address;
	}

	void Customers::setCustomersAddress(const std::wstring &value)
	{
		Address = value;
	}

	const int &Customers::getCustomersTellefon() const
	{
		return TelephonNumber;
	}

	void Customers::setCustomersTellefon(const int &value)
	{
		TelephonNumber = value;
	}

	void Customers::InitializeInstanceFields()
	{
		CustomerID = int();
		Name = std::wstring(NULL,20);
		FamilyName = std::wstring(NULL,20);
		Address = std::wstring(NULL,20);
		TelephonNumber = int();
	}
}
